source/guide_nextcloud.rst

```diff
@@ -100,6 +100,12 @@ PHP Memory
 
 Nextcloud `recommends 512 MB <https://docs.nextcloud.com/server/latest/admin_manual/installation/system_requirements.html>`_ of memory for an installation. To set the PHP memory limit to 512 MB, create the file ``~/etc/php.d/memory_limit.ini`` with the following content:
 
+.. code-block:: ini
+
+ memory_limit=512M
+ 
+If you still see an error in the web UI, edit ``~/etc/php.d/php.ini`` with the following content:
```

Comment:
It looks like that file only exist because you migrated from U6. You may add this to the troubleshooting section. The guide assumes to create a new U7 with default settings.

Reply:
Fine with me!
I mark this as a todo because I'm about to call it a day for today.

Reply:
Moved.